README.md

```diff
@@ -159,6 +159,39 @@ yarn build:integration
 yarn test:integration
 ```
 
+## Branching Model and Releases
+
+<!-- TODO: explain about changesets + how we do npm publishing + docker publishing -->
+
+### Active Branches
+
+| Branch          | Status                                                                           |
+| --------------- | -------------------------------------------------------------------------------- |
+| [master](https://github.com/ethereum-optimism/optimism/tree/master/)                   | Accepts PRs ONLY from `develop`                                                  |
+| [develop](https://github.com/ethereum-optimism/optimism/tree/develop/)                 | Accepts PRs that are compatible with `master` OR from `regenesis-X.X.X` branches |
+| [regenesis-0.4.0](https://github.com/ethereum-optimism/optimism/tree/regenesis-0.4.0/) | Deployed to Goerli + Kovan. **Branch frozen**. Merge to `develop` pending tests. |
```

Comment:
This will become stale very quickly - will you be responsible for updating it if you are adding such in depth information? Can you abstract this somehow behind a hyperlink to `contracts/deployments/README` so it updates automatically? If `hardhat deploy` keeps track of `git` repo info in the artifacts or something like that

Reply:
I'll just update it manually. We only need to update it once per release so should be OK. Writing myself a note.